iarc_arbiter/scripts/transformers.py

```diff
@@ -3,12 +3,15 @@
 from numpy import sign
 
 import rospy
+
 import tf.transformations
+from tf import TransformListener
+from tf.listener import xyz_to_mat44, xyzw_to_mat44
+
+from iarc_arbiter.msg import VelAlt
 from geometry_msgs.msg import Twist, PoseStamped, Pose, Point, Quaternion
 from nav_msgs.msg import Odometry
 from std_msgs.msg import Header
-from tf import TransformListener
-from tf.listener import xyz_to_mat44, xyzw_to_mat44
 
 
 class Command:
```

Comment:
Might be worthwhile grouping "takeoff" and "land" into "actions" and checking its parameter value, in case more actions of this kind are needed.